lib/main_common.pm

```diff
@@ -1749,7 +1749,8 @@ sub load_create_hdd_tests {
     # temporary adding test modules which applies hacks for missing parts in sle15
     loadtest 'console/sle15_workarounds' if is_sle('15+');
     loadtest 'console/integration_services' if is_hyperv;
-    loadtest 'console/hostname'       unless is_bridged_networking;
+    loadtest 'console/hostname' unless is_bridged_networking;
+    loadtest 'console/consoletest_setup' if any_desktop_is_applicable;
```

Comment:
Should we load it unconditionally, as need in case of textmode?

Reply:
Yes, it worked well in textmode. Because I had restricted using "any_desktop_is_applicable".